seeder/src/seeder_flet/lib/views/login.py

```diff
@@ -3,7 +3,7 @@
 __all__ = ["login_view"]
 
 
-async def handle_login_click(username: str, password: str, backend: str):
```

Comment:
Why was async removed?

Because to fetch data we need asyncio.nursery

Reply:
i think flet might not support asynchronous functions natively for handlers.
getting this error:
```bash
/home/tokitouq/Projects/CoreProject/seeder/.venv/lib/python3.12/site-packages/flet_core/page.py:528: RuntimeWarning: coroutine 'handle_login_click' was never awaited
  handler(*args)
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
```
https://github.com/flet-dev/flet/issues/128
https://flet.dev/blog/flet-mobile-update/#asyncio-support

or i think we can wrap it inside an `asyncio` function.
what do you think?

Reply:
something like:
```py
async def handle_login_click(username: str, password: str, backend: str):
    "Try to login with backend"
    ...

def handle_login_click_wrapper(e, username: str, password: str, backend: str):
    asyncio.run(handle_login_click(username, password, backend))
```